dbt_project.yml

```diff
@@ -22,4 +22,9 @@ vars:
   #   by_table_fqn: []
   # >> for metricflow
   # dbt_dq_tool_start_date
-  # dbt_dq_tool_end_date
\ No newline at end of file
+  # dbt_dq_tool_end_date
+
+models:
+  dq_tools:
+    +database: var('dbt_dq_tool_database', target.database),
+    +schema: var('dbt_dq_tool_schema', target.schema),
```

Comment:
Hi @rm684 Have you tested this yet?

```
cd (repo)/integration_tests
dbt deps
dbt run -s dq_tools
```
I think we need the curly brackets and without comma 🤔 
```yml
models:
  dq_tools:
    +database: "{{ var('dbt_dq_tool_database', target.database) }}
    +schema: "{{ var('dbt_dq_tool_schema', target.schema) }}"
```